src/Compatibility/Microsoft.DotNet.PackageValidation/ILogger.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    public interface ILogger
```

Comment:
NIT: How do you feel about using the same name as the abstraction MSBuild provides for this? I think it is not a big deal as this is in a different namespace, but I don't know if it might cause any problems in the future?